backend/db/migrate/20230608080932_rename_api_entreprise_scopes_in_enrollments.rb

```diff
@@ -0,0 +1,127 @@
+class RenameApiEntrepriseScopesInEnrollments < ActiveRecord::Migration[7.0]
+  def up
+    valid_enrollments.find_each do |enrollment|
+      scopes = enrollment.scopes.map do |scope|
+        case scope
+        when "associations"
+          "open_data"
+        when "associations_donnees_protegees"
+          "associations_djepva"
+        when "attestations_agefiph"
+          "attestations_agefiph"
+        when "attestations_fiscales"
+          "attestation_fiscale_dgfip"
+        when "attestations_sociales"
+          "attestation_sociale_urssaf"
+        when "bilans_entreprise_bdf"
+          "bilans_bdf"
+        when "fntp_carte_pro"
+          "open_data"
+        when "certificat_cnetp"
+          "certification_cnetp"
+        when "msa_cotisations"
+          "cotisations_msa"
+        when "certificat_opqibi"
+          "certification_opqibi"
+        when "probtp"
+          "cotisations_probtp"
+        when "qualibat"
+          "open_data"
+        when "certificat_rge_ademe"
+          "open_data"
+        when "exercices"
+          "chiffre_affaires_dgfip"
+        when "extrait_court_inpi"
+          "open_data"
+        when "extraits_rcs"
+          "open_data"
+        when "entreprises"
+          ["mandataires_sociaux_infogreffe", "unites_legales_etablissements_insee"]
```

Comment:
@skelz0r je me demande s'il ne manque pas open_data ici ?

```suggestion
          ["open_data", "mandataires_sociaux_infogreffe", "unites_legales_etablissements_insee"]
```